common/lib/common-utils/src/idleTaskScheduler.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryProperties } from "@fluidframework/common-definitions";
+
+/**
+ * API used to schedule non-essential tasks
+ * Time measurements are in milliseconds as a floating point with a decimal
+ */
+export class IdleTaskScheduler {
```

Comment:
Given that this class has no members (only one function), consider exporting flat function without any class